src/components/Results/Study.tsx

```diff
@@ -108,7 +103,31 @@ const Study = ({ entry, handleSaveStudy, isStudySaved }: StudyProps): ReactEleme
             {entry.contacts.map((contact, index) => (
               <StudyContact title="Contact" contact={contact} key={index} />
             ))}
-            <StudyContact title="Closest Facility" contact={entry.closestFacilities[0]} />
+            <Accordion
+              disableGutters
+              square
+              sx={{
+                marginTop: 2,
+                '&.MuiAccordion-root': { boxShadow: 'none' },
```

Comment:
I thought the gray borders on the white accordion were too jarring compared to the similarly light gray background of the Stack, so I removed them. But the blue buttons and other accordions do have some gray borders.